### ISSUE TITLE
Add License

### ISSUE THREAD

**Issue by kilrau** [bisq-network/bisq-website#429]
Even though the following should apply according to [GitHub Docs](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/licensing-a-repository#choosing-the-right-license):

> without a license, the default copyright laws apply, meaning that you retain all rights to your source code and no one may reproduce, distribute, or create derivative works from your work.

I am not sure if this the intention for the bisq website though. A e.g. `AGPL-3.0` license, which is also used for the bisq main application, seems more fitting. Would be great if someone from the team could clarify this (or just add a license).

**Comment by m52go** [comment]
Thanks for bringing this up -- never really thought about it. You're certainly right that the traditional restrictive copyright approach makes no sense to apply to the website. Just earlier today someone showed off a coffee mug with a picture from the website (which they intend to sell), and of course it was totally acceptable. 

But I'm also not sure a software license is appropriate for this website...it's more content than software, so maybe a publishing-style license like a Creative Commons would be a better fit? 

Main reason being that we categorically _don't_ want someone to host the website produced from this code on any other domain...whether they make any changes or not is irrelevant.

**Comment by kilrau** [comment]
Yep, makes sense. Why I brought this up: we @ [OpenDEX](https://opendex.network/) are in search for website source code we can use to host a website that allows users to download binaries for our [opendex-desktop](https://github.com/opendexnetwork/opendex-desktop) app. The bisq website does everything we'd need with some nice built-in features like recognizing the operating system.

Why not quickly code sth up ourselves? We will if I can't find anything reusable in the next couple of days, but we are a (very) small team, currently without funding and are trying to save development effort on everything that is not core development whenever possible.

So in short: this website would be hosted at `opendex-desktop.app` and (apart from a credit for the code) will have no content (binaries, texts, logos, pictures, screenshots, color scheme) or anything else to do with bisq.

Definitely understand though if you do not want the code to be used for this, so a quick `NACK`/`ACK` would do :v:

**Comment by m52go** [comment]
> will have no content (binaries, texts, logos, pictures, screenshots, color scheme) or anything else to do with bisq

So basically just using the Jekyll shell as a starting point for all new content, branding, etc...sounds reasonable to me :+1:

ACK

Closing since immediate issue is resolved, but may add copyright specification later.
